scan-command/src/main/java/io/ballerina/scan/utils/DiagnosticLog.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan.utils;
+
+import java.text.MessageFormat;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+/**
+ * Diagnostic log for scan tool.
+ *
+ * @since 0.1.0
+ */
+public class DiagnosticLog {
```

Comment:
Do you have tests for these errors?

Reply:
Yes, they have been added in the [DiagnosticLogTest class](https://github.com/ballerina-platform/static-code-analysis-tool/pull/15/files#diff-096e1e7f096c7da225cdf5c19778594d9e825bc369ac841798b8e41b793d7cc9)

Reply:
Yeah, but there we don't seem to have tests for all possible diagnostics?